### ISSUE TITLE
Releases `1.12.1` are not in PyPY

### ISSUE THREAD

**Issue by eirnym** [ninja-build/ninja#2530]
Please, publish latest Ninja versions to pypi

**Comment by eirnym** [comment]
There's specified ninja-build.org On page https://pypi.org/project/ninja/ as homepage which is quite misleading.

**Comment by idealseal** [comment]
I agree, this is misleading and should be addressed. The project is https://github.com/scikit-build/ninja-python-distributions. They base their releases in the Kitware ninja builds, which are a couple of releases behind.
